service/create/validation.go

```diff
@@ -0,0 +1,34 @@
+package create
+
+import (
+	"github.com/giantswarm/awstpr"
+	"github.com/giantswarm/awstpr/aws"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+func validateWorkers(workers []aws.Node) error {
+	if len(workers) < 1 {
+		return microerror.MaskAny(workersListEmptyError)
+	}
+
+	previousImageID := workers[0].ImageID
```

Comment:
Nitpick: this is not really *previous*, since you don't update it. It's rather *first*.